src/Java.Interop.Dynamic/Tests/Java.Interop.Dynamic-Tests.csproj

```diff
@@ -36,9 +36,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\lib\NUnit-2.6.3\bin\nunit.framework.dll</HintPath>
```

Comment:
It's probably getting the reference from one of the projects that are referenced like TestJVM.csproj with my change.

I'll redo this leaving the reference in but just fix the the path.